sleap/gui/commands.py

```diff
@@ -1863,15 +1864,24 @@ def delete_extra_skeletons(labels: Labels):
 
     @staticmethod
     def ask(context: CommandContext, params: dict) -> bool:
-
         filters = ["JSON skeleton (*.json)", "HDF5 skeleton (*.h5 *.hdf5)"]
-        filename, selected_filter = FileDialog.open(
-            context.app, dir=None, caption="Open skeleton...", filter=";;".join(filters)
-        )
+        template = (
+            context.app.skeletonTemplates.currentText()
+        )  # get selected template from dropdown
+        if template == "Custom":  # check whether to load from file or preset
+            filename, selected_filter = FileDialog.open(
+                context.app,
+                dir=None,
+                caption="Open skeleton...",
+                filter=";;".join(filters),
+            )
+        else:
+            filename = get_package_file(
+                f"sleap/skeletons/{template}.json"
+            )  # load from selected preset
```

Comment:
Nitpick: [Google style](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) comments and punctuation
```suggestion
        # Get selected template from dropdown
        template =  context.app.skeletonTemplates.currentText()
        
        # Check whether to load from file or preset
        if template == "Custom": 
            filename, selected_filter = FileDialog.open(
                context.app,
                dir=None,
                caption="Open skeleton...",
                filter=";;".join(filters),
            )
        else:  # Load from a selected preset
            filename = get_package_file(f"sleap/skeletons/{template}.json")
```